install.sh

```diff
@@ -15,10 +13,25 @@ TARGET=${TARGET-linux}
 libversion=3
 libVFile="./libversion"
 
+function has_cmd {
+	command -v "$1" >/dev/null 2>&1
+}
+
+function calculate_sha256 {
+	if has_cmd "sha256sum"; then
+		command sha256sum "$1" | cut -f1 -d" "
+	elif has_cmd "shasum"; then
```

Comment:
This is available on Linux too, so you can make this the first check and only test for `sha256sum` if missing.